posts/templates/posts/post_detail.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-xs-0 col-sm-3 col-md-4"></div>
+        <div class="col-xs-12 col-sm-6 col-md-4">
```

Comment:
This is the same width as on the profile. But as the font is larger here, it should indeed be wider.